### PR TITLE
Doc changes for v0.2.0

### PR DESCRIPTION
**Pull request by brmorr**
This is the replacement PR for #2, with just the doc content changes.  Note that this change doesn't actually rev a new 0.2.0 version.  That can be done after getting the content updated.

@joylashley I did go ahead make the changes you had requested on the new doc from #2.

### CODE REVIEW

**Comment by joylashley** [comment]
LGTM. Thanks, Brad!

**Comment by brmorr** [comment]
@ckedwards does everything look good from a technical pov regarding the description of authenticationType="credentials"?

File: documentation/docs/api/connectToServer.md [hunk 1]
```diff
@@ -24,6 +24,6 @@ Specify the URL of the SAS Viya server. This is the full context root, including
 
 Choose additional options for how to connect to the server. The following properties are supported:
 
-- `authenticationType: string`: Choose the method that will be used to authenticate requests to the SAS Viya server.
-  - Currently the only valid value is `"guest"`. This attribute is required to support a future update where
-    `"credential"` will be the default.
+- `authenticationType: string`: Choose the method to authenticate requests to the SAS Viya server.
+  - `'guest'` automatically signs in to the SAS Viya server as the guest user.
+  - `'credentials'` uses SAS Logon to establish an authenticated session.
```
Comment (ckedwards) [inline documentation/docs/api/connectToServer.md]: `authenticationType` is optional.

File: documentation/docs/api/registerDataDrivenContent.md [hunk 2]
```diff
@@ -23,9 +23,9 @@ Choose options for how to connect to the server. The following properties are su
   the protocol, optional port, and host.
 - `reportUri: string`: Specify the report URI.
 - `objectName: string`: Specify name of the data-driven content object.
-- `authenticationType: string`: Choose the method that will be used to authenticate requests to the SAS Viya server.
-  - Currently the only valid value is `"guest"`. This attribute is required to support a future update where
-    `"credential"` will be the default.
+- `authenticationType: string`: Choose the method to authenticate requests to the SAS Viya server.
+  - `'guest'` automatically signs in to the SAS Viya server as the guest user.
+  - `'credentials'` uses SAS Logon to establish an authenticated session.
```
Comment (ckedwards) [inline documentation/docs/api/registerDataDrivenContent.md]: This should be optional as well for constancy.

**Comment by brmorr** [comment]
Added default values for `connectToServer` and `registerDataDrivenContent`.

**Pull request merged**